### PR TITLE
Fix high priority tasks

### PR DESCRIPTION
**Pull request by barshathakuri**
## Changes
- Add Region 3W component
- Add icon for wildfire
- Add pending in login 
- Fix links in 3W

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/components/domain/RegionDropdown/i18n.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-    "namespace": "common",
+    "namespace": "regionDropdown",
```
Comment (tnagorra) [inline src/components/domain/RegionDropdown/i18n.json]:
```suggestion
    "namespace": "common",
```

File: src/views/GlobalThreeW/Map/i18n.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "namespace": "globalThreeWMap",
```
Comment (tnagorra) [inline src/views/GlobalThreeW/Map/i18n.json:2]:
```suggestion
    "namespace": "globalThreeW",
```

**Review by tnagorra — APPROVED** [review]

**Review by shreeyash07 — APPROVED** [review]

File: src/components/domain/RiskImminentEvents/index.tsx [hunk 3]
```diff
@@ -58,7 +59,11 @@ const riskHazards: Array<{
     {
         key: 'WF',
         label: 'Wildfire',
-        icon: null,
+        icon: <img
+            src={wildfireIcon}
+            className={styles.wildfireIcon}
+            alt="wildfire"
+        />,
```
Comment (frozenhelium) [inline src/components/domain/RiskImminentEvents/index.tsx]: We should use icon from the GO Icons

File: src/components/domain/Region3WDropdown/index.tsx
Comment (frozenhelium) [inline src/components/domain/Region3WDropdown/index.tsx:1]: This component can be localized to the Global 3W view

File: src/components/domain/Region3WDropdown/i18n.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "namespace": "region3WDropdown",
```
Comment (frozenhelium) [inline src/components/domain/Region3WDropdown/i18n.json]: Probably should use "threeW" namespace

**Review by frozenhelium — CHANGES_REQUESTED** [review]

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**